docs/run_publish/references.md

```diff
@@ -166,17 +156,29 @@ There are 7 options to choose from. “fatal”, “error”, “warn”, “inf
 
 ### --multi-chain
 
-Enables indexing multiple subquery projects into the same database schema.
+**Boolean** - Enables indexing multiple subquery projects into the same database schema.
 
 ```shell
 > subql-node -f . --multi-chain --db-schema=SCHEMA_NAME
 ```
 
 For more info, visit [Multi-Chain Support](../build/multi-chain.md).
 
+### -d, --network-dictionary
+
+**String (default: Network dictionary from your manifest)** - This allows you to specify a dictionary GraphQL endpoint which is a free service that is provided and hosted at SubQuery's [Project Explorer](https://explorer.subquery.network/). You can search for dictionary and and enter the GraphQL API endpoint (e.g. https://api.subquery.network/sq/subquery/dictionary-polkadot).
+
+Typically this would be set in your manifest file but below shows an example of using it as an argument in the command line.
+
+```shell
+subql-node -f . -d "https://api.subquery.network/sq/subquery/dictionary-polkadot"
+```
+
+For more info, visit [How does a SubQuery Dictionary works?](../academy/tutorials_examples/dictionary.md)
+
 ### --network-endpoint
 
-This flag allows users to override the network endpoint configuration from the manifest file.
+**String (default: Network endpoint from your manifest)** - This flag allows users to override the network RPC API endpoint configuration from the manifest file.
```

Comment:
Is it worth mentioning we can repeat these? `subql-node -f ./project.yaml --network-endpoint=<endpoint-1> --network-endpoint=<endpoint-2>`